apps/server/src/modules/class/service/class.service.spec.ts

```diff
@@ -148,18 +157,58 @@ describe(ClassService.name, () => {
 
 			it('should call classesRepo.findAllByUserId', async () => {
 				const { userId1 } = setup();
-				await service.deleteUserDataFromClasses(userId1.toHexString());
+				await service.deleteUserData(userId1.toHexString());
 
 				expect(classesRepo.findAllByUserId).toBeCalledWith(userId1.toHexString());
 			});
 
 			it('should update classes without updated user', async () => {
 				const { expectedResult, userId1 } = setup();
 
-				const result = await service.deleteUserDataFromClasses(userId1.toHexString());
+				const result = await service.deleteUserData(userId1.toHexString());
 
 				expect(result).toEqual(expectedResult);
 			});
 		});
 	});
+
+	describe('handle', () => {
+		const setup = () => {
+			const targetRefId = new ObjectId().toHexString();
+			const targetRefDomain = DomainName.CLASS;
+			const classId = new ObjectId().toHexString();
+			const deletionRequest = deletionRequestFactory.build({ targetRefId, targetRefDomain });
+
+			const expectedData = DomainDeletionReportBuilder.build(DomainName.CLASS, [
+				DomainOperationReportBuilder.build(OperationType.UPDATE, 1, [classId]),
+			]);
+
+			return {
+				deletionRequest,
+				expectedData,
+			};
+		};
+
+		describe('when UserDeletedEvent', () => {
```

Comment:
Please add some word or two about the circumstances here (when _what_ happens exactly?).

Reply:
E.g. "when UserDeletedEvent is received" or something like similar that would give any background to anyone reading that in the future.

Reply:
done